riscv-sbi.adoc

```diff
@@ -160,6 +154,10 @@ returning an error code. This is analogous to returning the C structure
     };
 ----
 
+In the name of compatibility, SBI extension IDs (*EIDs*) and SBI function IDs
+(*FIDs*) are encoded as signed 32-bit integers. When passed in registers these
```

Comment:
- say "should" instead of "are"? New EIDs/FIDs may exceed this range and older software will never use them?